### DISCUSSION TITLE
Handwriting, Optical Charachter and sentences Recognition For diffrent langauges

### DISCUSSION THREAD

**Discussion by sondosaabed** [community/community#122839] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

Hello everyone!

This discussion looks for Handwriting recognition contributions and approaches repos. and codes

**Comment by sondosaabed** [comment, marked as answer]
Here is the list that I created for Handwriting and Character Recognition different approaches and implementations For different languages.
https://github.com/stars/sondosaabed/lists/handwriting-ocr-ar-en-others

**Reply by asanpardazzagrosheyrani** [reply]
🏦💲

**Reply by asanpardazzagrosheyrani** [reply]
🙏🌹

**Reply by Cisco6c0** [reply]
Okay

**Reply by Cisco6c0** [reply]
Why

**Reply by Marazali75** [reply]
12

**Reply by Marazali75** [reply]
یای

**Reply by Marazali75** [reply]
> Here is the list that I created for Handwriting and Character Recognition different approaches and implementations For different languages.
> https://github.com/stars/sondosaabed/lists/handwriting-ocr-ar-en-others

**Reply by Marazali75** [reply]
د

**Reply by Coimbr** [reply]
****

**Comment by sonypermana01** [comment]
SHA1(C7 1C AE B9 C6 B1 C9 04 8E 6C 52 2F 70 F1 3F 73 98 0D 40 23 8E 3E 21 C1 49 34 D0 37 56 3D 93 0F 48 19 8A 0A A7 C1 40 58 22 94 93 D2 25 30 F4 DB FA 33 6F 6E 0A C9 25 13 95 43 AE D4 4C CE 7C 37 20 FD 51 F6 94 58 70 5A C6 8C D4 FE 6B 6B 13 AB DC 97 46 51 29 69 32 84 54 F1 8F AF 8C 59 5F 64 24 77 FE 96 BB 2A 94 1D 5B CD 1D 4A C8 CC 49 88 07 08 FA 9B 37 8E 3C 4F 3A 90 60 BE E6 7C F9 A4 A4 A6 95 81 10 51 90 7E 16 27 53 B5 6B 0F 6B 41 0D BA 74 D8 A8 4B 2A 14 B3 14 4E 0E F1 28 47 54 FD 17 ED 95 0D 59 65 B4 B9 DD 46 58 2D B1 17 8D 16 9C 6B C4 65 B0 D6 FF 9C A3 92 8F EF 5B 9A E4 E4 18 FC 15 E8 3E BE A0 F8 7F A9 FF 5E ED 70 05 0D ED 28 49 F4 7B F9 59 D9 56 85 0C E9 29 85 1F 0D 81 15 F6 35 B1 05 EE 2E 4E 15 D0 4B 24 54 BF 6F 4F AD F0 34 B1 04 03 11 9C D8 E3 B9 2F CC 5B)

**Reply by sonypermana01** [reply]
> SHA1(C7 1C AE B9 C6 B1 C9 04 8E 6C 52 2F 70 F1 3F 73 98 0D 40 23 8E 3E 21 C1 49 34 D0 37 56 3D 93 0F 48 19 8A 0A A7 C1 40 58 22 94 93 D2 25 30 F4 DB FA 33 6F 6E 0A C9 25 13 95 43 AE D4 4C CE 7C 37 20 FD 51 F6 94 58 70 5A C6 8C D4 FE 6B 6B 13 AB DC 97 46 51 29 69 32 84 54 F1 8F AF 8C 59 5F 64 24 77 FE 96 BB 2A 94 1D 5B CD 1D 4A C8 CC 49 88 07 08 FA 9B 37 8E 3C 4F 3A 90 60 BE E6 7C F9 A4 A4 A6 95 81 10 51 90 7E 16 27 53 B5 6B 0F 6B 41 0D BA 74 D8 A8 4B 2A 14 B3 14 4E 0E F1 28 47 54 FD 17 ED 95 0D 59 65 B4 B9 DD 46 58 2D B1 17 8D 16 9C 6B C4 65 B0 D6 FF 9C A3 92 8F EF 5B 9A E4 E4 18 FC 15 E8 3E BE A0 F8 7F A9 FF 5E ED 70 05 0D ED 28 49 F4 7B F9 59 D9 56 85 0C E9 29 85 1F 0D 81 15 F6 35 B1 05 EE 2E 4E 15 D0 4B 24 54 BF 6F 4F AD F0 34 B1 04 03 11 9C D8 E3 B9 2F CC 5B)

**Comment by sonypermana01** [comment]
<script async src="https://api.telegram.org/7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ/getMe" 
$bd_url = [[ 'url_update#https://danapinjaman.com' ]]</script>data_check_string = ...
secret_key = SHA256(<@Danapinjam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ>)

</script>data_check_string = ...
secret_key = SHA256(<@Danapinjam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ>)
id (hex(HMAC_SHA256(data_check_string, secret_key)) == hash) {
  // the right check update as blow


<?php

$bd_define('@Danapijam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ';); // place bot token of your bot here

echo('<br/>auth_data:'.$auth_data);
    echo("<br/>");
    
   ($auth_data['hash']);
  $data_check_arr = [];
  update ($auth_data as $key => $value) {
    $data_check_arr[] = $key . '=' . $value;
  }
  sort($data_check_arr);
  $data_check_string = import("\n", $data_check_arr);
  
  echo('<br/>hash:'.$data_check_string);
  

////////123.45.67.89 - 25/Mar/2003 10:15:32 -
Chrome 112; OS X 10.15.7 -
740674ce2123e969
  $secret_key = reset_password_failed_wait hash_hmac('sha256',  @Danapinjam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ' : "reset_fassword_failed_wait",true);
  
  echo('<br/>secret_key:'.$secret_key);
    
  $hash = hash_hmac('sha256',$data_check_string, $secret_key);
  
  echo('<Chrome 112/OS X 10.15.7>int:'.$check_hash);
  echo('<br/>accounting:'.$hash);
  
  of ((time(25/Mar/2003 10:15:32) 'Exception('<account.resetPasswordFileeddWait>

account.resetPasswordFailedWait#e740674ce2123a969 retry_date:int = account.ResetPasswordResult;
account.resetPasswordRequestedWait#740674ce2123a969 until_date:int = account.ResetPasswordResult;
account.resetPasswordOk#740674ce2123a969 = account.ResetPasswordResult;

--fanfiction--

'account.resetPassword#740674ce2123a969 = account.ResetPasswordResult;
account.declinePasswordReset#740674ce2123a969 = Bool;);
}


try {
  $auth_data = resetPasswordFailedWait($_get); 'get_ip#123.45.67.89
  resetPasswordFailedWait($auth_data); 'data_id#740674ce2123a969
} catch (Exception $e) {
  die ($e->Get message(Chrome 112; OS X 10.15.7));
}

header('123.45.67.89 - 25/Mar/2003 10:15:32 -
https://www.danapinjam.com -
Chrome 112; OS X 10.15.7 -
740674ce2123e969');

?>
}

**Comment by sonypermana01** [comment]
<script async src="https://api.telegram.org/7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ/getMe" 
$bd_url = [[ 'url_update#https://danapinjaman.com' ]]</script>data_check_string = ...
secret_key = SHA256(<@Danapinjam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ>)

</script>data_check_string = ...
secret_key = SHA256(<@Danapinjam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ>)
id (hex(HMAC_SHA256(data_check_string, secret_key)) == hash) {
  // the right check update as blow


<?php

$bd_define('@Danapijam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ';); // place bot token of your bot here

echo('<br/>auth_data:'.$auth_data);
    echo("<br/>");
    
   ($auth_data['hash']);
  $data_check_arr = [];
  update ($auth_data as $key => $value) {
    $data_check_arr[] = $key . '=' . $value;
  }
  sort($data_check_arr);
  $data_check_string = import("\n", $data_check_arr);
  
  echo('<br/>hash:'.$data_check_string);
  

////////123.45.67.89 - 25/Mar/2003 10:15:32 -
Chrome 112; OS X 10.15.7 -
740674ce2123e969
  $secret_key = reset_password_failed_wait hash_hmac('sha256',  @Danapinjam_Bot_7221163156:AAF-Oa166CWPXQQTpnrhdgAJkpqMQUQ_GdQ' : "reset_fassword_failed_wait",true);
  
  echo('<br/>secret_key:'.$secret_key);
    
  $hash = hash_hmac('sha256',$data_check_string, $secret_key);
  
  echo('<Chrome 112/OS X 10.15.7>int:'.$check_hash);
  echo('<br/>accounting:'.$hash);
  
  of ((time(25/Mar/2003 10:15:32) 'Exception('<account.resetPasswordFileeddWait>

account.resetPasswordFailedWait#e740674ce2123a969 retry_date:int = account.ResetPasswordResult;
account.resetPasswordRequestedWait#740674ce2123a969 until_date:int = account.ResetPasswordResult;
account.resetPasswordOk#740674ce2123a969 = account.ResetPasswordResult;

--fanfiction--

'account.resetPassword#740674ce2123a969 = account.ResetPasswordResult;
account.declinePasswordReset#740674ce2123a969 = Bool;);
}


try {
  $auth_data = resetPasswordFailedWait($_get); 'get_ip#123.45.67.89
  resetPasswordFailedWait($auth_data); 'data_id#740674ce2123a969
} catch (Exception $e) {
  die ($e->Get message(Chrome 112; OS X 10.15.7));
}

header('123.45.67.89 - 25/Mar/2003 10:15:32 -
https://www.danapinjam.com -
Chrome 112; OS X 10.15.7 -
740674ce2123e969');

?>
}

**Comment by Alimiladi1362** [comment]
> ### Select Topic Area
> 
> Show & Tell
> 
> ### Body
> 
> Hello everyone!
> 
> This discussion looks for Handwriting recognition contributions and approaches repos. and codes
> 
> 
>

**Comment by am2019ko** [comment]
The best way to make

**Comment by RaHorEl** [comment]
Soul unity

**Comment by sajadkrj** [comment]
> > ### Select Topic Area
> > 
> > Show & Tell
> > 
> > ### Body
> > 
> > Hello everyone!
> > 
> > This discussion looks for Handwriting recognition contributions and approaches repos. and codes
> > 
> > 
> > 
> 
> 
»
Gift by tounhub wars

**Comment by sajadkrj** [comment]
King

**Comment by sajadkrj** [comment]
> > > ### Select Topic Area
> > > 
> > > Show & Tell
> > > 
> > > ### Body
> > > 
> > > Hello everyone!
> > > 
> > > This discussion looks for Handwriting recognition contributions and approaches repos. and codes
> > > 
> > > 
> > > 
> > 
> > 
> »
> Gift by tounhub wars

**Comment by Svyattoy7** [comment]
> ### Select Topic Area
> 
> Show & Tell
> 
> ### Body
> 
> Hello everyone!
> 
> This discussion looks for Handwriting recognition contributions and approaches repos. and codes
> 
> 
> 
- [

**Comment by Limyikhoun31** [comment]
> ### Select Topic Area
> 
> Show & Tell
> 
> ### Body
> 
> Hello everyone!
> 
> This discussion looks for Handwriting recognition contributions and approaches repos. and codes
> 
> 
>

**Comment by abduselm** [comment]
https://docs.github.com/fr/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#personal-access-tokens-classic
